### DISCUSSION TITLE
Transition not working react 18

### DISCUSSION THREAD

**Discussion by luuhung93** [tailwindlabs/headlessui#1309] [General]
Transition not working react 18,
Expect, fix in next update.

**Comment by lveillard** [comment]
It's not working in development when react strict is set to true, but it should be working on production. 
You can disable strict mode temporally while they fix it

**Reply by cavellblood** [reply]
How do you disable strict mode in the development environment? I searched my codebase for `strict` and couldn't find it anywhere but the transitions are not working on my project just like @luuhung93 mentioned. When I remove the transition wrapping element then things show up.

**Reply by RobinMalfait** [reply]
Please checkout https://github.com/tailwindlabs/headlessui/issues/681 for more info

**Reply by cavellblood** [reply]
Thanks @RobinMalfait. I've ran `yarn add @headlessui/react@insiders` and that fixed the transition issue.

**Comment by luuhung93** [comment]
https://reactjs.org/docs/strict-mode.html
https://nextjs.org/docs/api-reference/next.config.js/react-strict-mode
